Digital_Clock/digital_clock.py

```diff
@@ -0,0 +1,56 @@
+
+from tkinter import *
```

Comment:
Please import external libraries/packages using aliases instead of importing all of the content using `*`.  Since this is a small project, It can be acceptable but please do remember to follow best practices in your further contribution.

Making **ONLY** the below suggested change may cause the whole source code to break. So, please respective changes to your source code if you wish to update your code with following changes.

```suggestion
import tkinter as tk
```